metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -215,6 +216,11 @@ public Builder setAlterConfigPolicy(Optional<AlterConfigPolicy> alterConfigPolic
             return this;
         }
 
+        public Builder setConfigurationValidator(ConfigurationValidator configurationValidator) {
+            this.configurationValidator = configurationValidator;
+            return this;
+        }
```

Comment:
This method is never called in this PR. Are we missing changes?

Reply:
Yes, there was supposed to be a ControllerConfigurationValidator file. Added.